sshd-core/src/test/resources/org/apache/sshd/client/keyverifier/known_hosts

```diff
@@ -13,3 +13,5 @@ server.sshd.apache.org,10.23.222.240 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbml
 
 # non-standard port overrides
 [issues.apache.org]:5637 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBCWDHD00Ltb5fmmL9cFLTqWqxgJHwsxbiZgL632CXqbDmf69wA+8GSP96rtIix2d5aGXyh/kXMbSMjPgIx+n7p0=
+
+fe80::7780:db3:a57:6a9 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBCWDHD00Ltb5fmmL9cFLTqWqxgJHwsxbiZgL632CXqbDmf69wA+8GSP96rtIix2d5aGXyh/kXMbSMjPgIx+n7p0=
```

Comment:
Note that it does not cover [link-local](https://en.wikipedia.org/wiki/IPv6#Link-local_address) address, or [scoped address](https://en.wikipedia.org/wiki/IPv6_address) - but for now, it should be good enough